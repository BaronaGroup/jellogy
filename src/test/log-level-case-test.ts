import {prepareTestKarhu} from './test-utils'

describe('default-log-levels-test', () => {
  const karhuTest = prepareTestKarhu({})

  it('specified case and lowercase calls work', karhuTest((karhu, output) => {
    const ctx = karhu.context('ctx')
    ctx.ERROR('error1')
    ctx.error('error2')
    expect(output.tracked).toMatchSnapshot()
  }))
})
