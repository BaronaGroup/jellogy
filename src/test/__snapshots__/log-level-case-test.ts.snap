// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default-log-levels-test specified case and lowercase calls work 1`] = `
Map {
  "NONE" => Array [],
  "DEBUG" => Array [],
  "INFO" => Array [],
  "WARN" => Array [],
  "ERROR" => Array [
    Array [
      "0 ERROR ctx[97m[101m",
      "error1",
      "[49m[39m",
    ],
    Array [
      "1 ERROR ctx[101m[97m",
      "error2",
      "[39m[49m",
    ],
  ],
}
`;
